float64 velocityRight
float64 velocityLeft
---
bool response
